lxd/device/disk.go

```diff
@@ -74,7 +74,7 @@ type diskSourceNotFoundError struct {
 }
 
 func (e diskSourceNotFoundError) Error() string {
-	return fmt.Sprint(e.msg, ": ", e.err)
+	return fmt.Sprintf("%s: %v", e.msg, e.err)
```

Comment:
This was changed just 5 days ago by @simondeziel in https://github.com/canonical/lxd/commit/fcf892a6d8801a53a972d0a241433cf68be1b313 to avoid using sprintf that uses reflection.